CHANGELOG.rst

```diff
@@ -1,6 +1,18 @@
 Changelog
 ---------
 
+1.0.0 (unreleased)
+++++++++++++++++++
+
+Features:
+
+- References nested Schemas by adding the Schema to the spec with an automatically
```

Comment:
How about

```
- Automatically generate references to nested schemas with a computed name, e.g.
  ``fields.Nested(PetSchema())`` -> ``#components/schemas/Pet``.
- Automatically generate references for ``requestBody`` using the above mechanism.
```

Make sure to add periods after each item for consistency with the rest of the changelog.